app-name/src/NavBar.jsx

```diff
@@ -0,0 +1,21 @@
+export default function NavBar({ onClick }) {
```

Comment:
I think a name more specific to what was clicked would make the code more understandable. Something like `onNav` or `onEntries`?